docs/advanced-entity-analytics/analyze-risk-score-data.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[analyze-risk-score-data]]
+== View and analyze risk score data
+
+The {security-app} provides several options to monitor the change in the risk posture of hosts and users from your environment. We recommend that you prioritize alert triaging to identify anomalies or abnormal behavior patterns.
+
+Use the following places in the {security-app} to view and analyze risk score data:
+
+[discrete]
+=== Entity Analytics dashboard
+
+From the Entity Analytics dashboard, you can access entity key performance indicators (KPIs), risk scores, and levels. You can also click the number link in the **Alerts** column to investigate and analyze the alerts on the Alerts page.
+
+[role="screenshot"]
+image::images/ea-dashboard.png[Entity Analytics dashboard]
+
+[discrete]
+=== Alerts page
+
+Use the Alerts table to investigate and analyze host and user risk levels and scores.
+
+TIP: You can <<customize-the-alerts-table, customize the Alerts table>> by adding or reordering columns to easily visualize the relevant data.
```

Comment:
We should probably mention the specific field/column (`user.risk.calculated_level`) since it's hard to read tiny text in the screenshot. It's mentioned below, but only in context of the drop-down filters, not the table.

Also, from the Tip I'm getting the sense that this column isn't displayed by default — if so, we directly tell users they need to do this.